Library/Homebrew/cask/lib/hbc/cli/doctor.rb

```diff
@@ -116,6 +117,21 @@ def check_environment_variables
         (locale_variables + environment_variables).sort.each(&method(:render_env_var))
       end
 
+      def check_quarantine_support
+        ohai "Gatekeeper support"
+
+        status = Quarantine.check_quarantine_support
+
+        case status
+        when :no_swift
+          add_error "Swift is not available on this system."
+        when :no_quarantine
+          add_error "This feature requires the macOS 10.10 SDK or higher."
+        else
```

Comment:
I'd like to see:

* a `when :quarantine_available` for enabled instead of a default case
* a `fail "unknown quarantine status"` in the default case

This guards against `Quarantine.check_quarantine_support` changing its return.